js2.js

```diff
@@ -0,0 +1,20 @@
+"use script"
+
+let t=1;
```

Comment:
tip: before and after operators + , - , * , =, etc. use spaces for better code formatting